contracts/files/FileInformation.ts

```diff
@@ -0,0 +1,39 @@
+import {Timestamp} from 'firebase/firestore';
+import {z} from 'zod';
+
+/**
+ * @interface FileInformation
+ * @description Represents the information of a file.
+ *
+ * @property {string} name - The name of the file.
+ * @property {string} extension - The extension of the file.
+ * @property {number} size - The size of the file in bytes.
+ * @property {Timestamp} uploaded_at - The timestamp when the file was uploaded.
+ *
+ * @since 1.0.0
+ */
+
+export interface FileInformation {
+    name: string;
+    extension: string;
+    size: number;
+    uploaded_at: Timestamp;
+}
+
+export const FileInformationSchema: z.ZodSchema<FileInformation> = z.object({
+  name: z.string({
+    required_error: 'File name is required',
+    invalid_type_error: 'File name must be a string',
+  }),
+  extension: z.string({
+    required_error: 'File extension is required',
```

Comment:
context of use? if all type of file, maybe can drop require because there is files without extension

Reply:
It should require an extension. We shouldn't keep files without an extension, it's not safe.

Reply:
FYI, if we save a file without an extension in FB storage, it becomes a folder.